### PR TITLE
Roles formatting

### PR DESCRIPTION
**Pull request by dcwalk**
This should be merged after #115 and #116 (and updated if there are changes there)

### CODE REVIEW

File: roles.md [hunk 1]
```diff
@@ -18,24 +18,33 @@ The *Cultivator* is responsible for tending to the all hands meetings and import
...
+**Authority:**
+- set agenda and determine discussion items for all hands
+
+**Limits:** (on Authority)
+- permission of member-workers needed to:
+	- cancel or reschedule call for week
+
+**Time Budget:** Up to 2 hours/week.
+
+**Length and Succession**: 1 month, rotates through member-workers. Last meeting of each month, the Cultivator asks for volunteers for the next person, if no one volunteers they can request those who have not been one recently to do so, which can be declined within based on discussion and with cause.
```
Comment (patcon) [inline roles.md:42]: maybe not the place to decide, but with succession, feels like something missing about knowing where to find the "big arc" of who's done a role. I feel like we lose that context when the ask is made for new ppl to step up. Might be nice to have one clear place, and I might like that that mentioned here if/once it exists.

But that's perhaps work for later, and no block.

**Review by patcon — APPROVED** [review]
LGTM 🌮

**Comment by patcon** [comment]
"looks good taco me" apparently 🤷

File: roles.md (hunk 1 above)
Comment (dcwalk) [reply roles.md:42]: At this point that info would be contained in WGs history, but yeah, something to ponder. I think to address later tho

**Pull request merged**